go.mod

```diff
@@ -4,6 +4,7 @@ go 1.17
 
 require (
 	github.com/conduitio/conduit-connector-sdk v0.2.0
+	github.com/google/uuid v1.1.2
```

Comment:
We should use the latest version though, `v1.3.0`

Reply:
I didn't check tbh, I thought it would automatically pull the latest version :S

Reply:
I think `go get` fetches the latest version, unless you already have one locally, then it takes that one. You can use `go get my-dependency@latest` to always fetch the latest.